pkgs/development/python-modules/certifi/env.patch

```diff
@@ -51,37 +51,3 @@ index 91f538b..1110ce0 100644
  
      def where() -> str:
          # This is slightly terrible, but we want to delay extracting the
-@@ -80,7 +92,9 @@ elif sys.version_info >= (3, 7):
```

Comment:
https://github.com/certifi/python-certifi/commit/d7c816cbc9cf621b3ddc8cd7fa7eda3f36982620 we no longer need that